forum/communications/serializers.py

```diff
@@ -1,10 +1,72 @@
 from rest_framework import serializers
+from .mongo_models import Room
+from .helpers import is_namespace_info_correct
+from bson.objectid import ObjectId
+from bson.errors import InvalidId
+
+
+class NamespaceInfoSerializer(serializers.Serializer):
+    user_id = serializers.IntegerField(required=True)
+    namespace = serializers.ChoiceField(choices=[
+        ("startup","startup"),
+        ("investor","investor")
+    ], required=True)
+    namespace_id = serializers.CharField(required=True)
+
+
+class RoomSerializer(serializers.Serializer):
+    participants = serializers.ListField()
+
+    def validate(self, data):
+        participants = data.get("participants")
+
+        if len(participants) != 2:
+            raise serializers.ValidationError("Only two participants in one room.")
+
+        for participant in participants:
+            if not NamespaceInfoSerializer(data=participant).is_valid():
+                raise serializers.ValidationError("Invalid participant.")
+            
+            namespace_obj = is_namespace_info_correct(participant)
+            
+            if not namespace_obj:
+                raise serializers.ValidationError("Incorrect participants data.")
+        
+        namespaces = [p.get("namespace") for p in participants]
+
+        if ("startup" not in namespaces) or ("investor" not in namespaces):
+            raise serializers.ValidationError("Room can be created only for investor and startup.")
+
+        return data
 
 
 class ChatMessageSerializer(serializers.Serializer):
```

Comment:
The `type` field should be defined, cause this serializer will be used by `communication.utils.AutoSerializer` (reason: determination (using `type` field) which serializer should be applied to validate ws message)
```Python
    type = serializers.CharField(required=True)
```